src/styles/index.sass

```diff
@@ -0,0 +1,29 @@
+
+// ## Mixins
+// Bourbon docs at http://bourbon.io/docs/
+@import '../../node_modules/bourbon/app/assets/stylesheets/bourbon'
+
+// ## Settings and variables
+@import 'variables'
+
+// ## Animations
+@import '_animations'
```

Comment:
V SASS se tak označují partials a tady nám to pěkně odděluje hlavní soubory od vedlejších. Ještě bychom mohli přejmenovat další, ať je to patrné.

Nicméně komponenty a další soubory ve složkách už tak dělat není potřeba.